extensions/amp-story/validator-amp-story.protoascii

```diff
@@ -258,6 +258,38 @@ tags: {
   attrs: { name: "animate-in-delay" }
   attrs: { name: "animate-in-duration" }
   attrs: { name: "grid-area" }
+  attrs: {
+    name: "scale-end"
+    value_regex: "^[0-9]*$"
```

Comment:
I think you may want to reconsider this regular expression (here and below); as it's written it will:

- Allow empty string
- Allow 0 (maybe that's ok, even though it's kind of weird)
- Disallow decimals

Reply:
Note also that the "^" and "$" are implied. This behavior varies between different languages and implementations (javascript it's not implied), so we normalize in the validator code. You can assume that value_regex is a full match and blacklisted_value_regex is a partial match (ie: in both cases, the least likely to match or most conservative).

Reply:
@newmuis I revised the regex, let me know what you think.

@Gregable thanks for the explanation!